packages/core/src/amazonqGumby/chat/controller/controller.ts

```diff
@@ -334,6 +340,24 @@ export class GumbyController {
         }
     }
 
+    private async handleSkipTestsSelection(message: any) {
+        const skipTestsSelection = message.formSelectedValues['GumbyTransformSkipTestsForm']
+        // one of these must be true
+        if (skipTestsSelection === CodeWhispererConstants.skipUnitTestsMessage) {
+            transformByQState.setCustomBuildCommand(CodeWhispererConstants.skipUnitTestsBuildCommand)
+        } else if (skipTestsSelection === CodeWhispererConstants.doNotSkipUnitTestsMessage) {
```

Comment:
Is it worth just making this an `else` and not skipping unit tests by default? What happens if `setCustomBuildCommand` is never called?

Reply:
Yes, will fix. Our default should be to not skip tests.